system/lib/wasmfs/syscalls.cpp

```diff
@@ -288,6 +288,45 @@ __wasi_errno_t __wasi_fd_close(__wasi_fd_t fd) {
   return __WASI_ERRNO_SUCCESS;
 }
 
+// This function is exposed to users to allow them to obtain a backend_t for a
+// specified path.
+backend_t wasmfs_get_backend(char* path) {
+  auto pathParts = splitPath(path);
+
+  if (pathParts.empty()) {
+    return nullptr;
+  }
+
+  // TODO: Remove this when path parsing has been re-factored.
```

Comment:
When do we plan to do this? This pattern seems to be very common.

Reply:
I was going to do this after we had implemented the JS Backend and Proxying Backend.